Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -12,7 +12,7 @@ import CoreLocation
 
 class HomeModelController: HomeViewController {
     
-    private var isTesting = true
```

Comment:
If this testing variable is no longer needed, you can consider removing it and any associated logic from the vc.